modules/ingestor/src/graph/advisory/advisory_vulnerability.rs

```diff
@@ -31,7 +31,7 @@ pub enum Version {
 }
 
 impl VersionInfo {
-    fn into_active_model(self) -> version_range::ActiveModel {
+    pub fn into_active_model(self) -> version_range::ActiveModel {
```

Comment:
bonus points: if you can understand why implementing `IntoActiveModel` caused rustc to complain about duplicate impls, that'd be lovely. There was a trait I could use, but I couldn't make rustc happy, hence this one-off fn.

Reply:
That one was a bit weird, it seems to be coming from the blanket implementation of that trait in seaorm:

```rust
impl<A> IntoActiveModel<A> for A
where
    A: ActiveModelTrait,
{
    fn into_active_model(self) -> A {
        self
    }
}
```

Which basically means that you can't implement `IntoActiveModel` for anything that should become an `ActiveModel`. Which feels like an API bug to me.

It also doesn't seem possible to implement `From` or `Into` for that case due to similar conflicts.

Reply:
Ok, that's not correct. Because in this case `A` would be `VersionInfo`, which doesn't implement `ActiveModelTrait`. And the compiler complains about `impl sea_orm::IntoActiveModel<trustify_entity::version_range::ActiveModel> for <trustify_entity::version_range::Entity as sea_orm::EntityTrait>::Model` being the conflict.

Reply:
Ok, I give up on the bonus points and leave it a mystery.